src/Admin.js

```diff
@@ -9,9 +9,9 @@ import adminReducer from './reducer';
 import crudSaga from './sideEffect/saga';
 import CrudRoute from './CrudRoute';
 import Layout from './mui/layout/Layout';
-import withAppTitle from './mui/layout/withAppTitle';
+import withAppTitleAndTheme from './mui/layout/withAppTitleAndTheme';
 
-const Admin = ({ restClient, dashboard, children, title = 'Admin on REST', appLayout = withAppTitle(title)(Layout) }) => {
+const Admin = ({ restClient, dashboard, children, title = 'Admin on REST', theme = {}, appLayout = withAppTitleAndTheme(title, theme)(Layout) }) => {
```

Comment:
With `{}` the default theme will be used. `getMuiTheme()` expects an object containing the values you would like to override. Hence passing an empty object, nothing will be overridden.